src/pages/Chat/ChatConversation.tsx

```diff
@@ -29,6 +32,7 @@ export const ChatConversation: React.SFC = () => (
       goToStep,
       livingSituation,
       currentInsurance,
+      nameAge,
```

Comment:
What is a nameAge? Is it the combined input of first name, last name and age?

Reply:
@frapperino see `state.tsx`:
```ts
{
  visibleSteps: [ChatStep.INITIAL],
  currentStep: ChatStep.INITIAL,
  initialVisibleSteps: [],
  nameAge: {
    firstName: '',
    age: '',
    lastName: '',
  },
  livingSituation: {
    size: 0,
    numberOfPeople: 0,
    postalNumber: '',
    streetAddress: '',
  },
  currentInsurance: {},
}```